### DISCUSSION TITLE
How to change in IOS drop-in  when re-using card the name of the app

### DISCUSSION THREAD

**Discussion by YankovWeb** [Adyen/adyen-react-native#390] [Q&A] [answered]
![Screenshot at Mar 22 11-44-43](https://github.com/Adyen/adyen-react-native/assets/109748669/6a492bbc-a260-4999-b069-7b8a2196d98e)
how use custom text ?

**Comment by descorp** [comment, marked as answer]
This is indeed missing scenario.
Currently there is no way to override this. Created issue #391  

Currently this value is calculated as:
```swift
private extension Bundle {
    var displayName: String {
        object(forInfoDictionaryKey: "CFBundleDisplayName") as? String ??
            object(forInfoDictionaryKey: "CFBundleName") as? String ?? ""
    }
}
```

**Reply by YankovWeb** [reply]
Thank you very much!

**Reply by descorp** [reply]
Config for DropIn title released on [2.0.0-rc.1](https://github.com/Adyen/adyen-react-native/releases/tag/2.0.0-rc.1)

use [configuration](https://github.com/Adyen/adyen-react-native/blob/develop/docs/Configuration.md#drop-in):

```js
dropin: {
    title: "my cool title"
}
```
